### PR TITLE
Fix dark monitor database typo

### PR DESCRIPTION
**Pull request by bhilbert4**
Two small typos fixed here. The first fixes a bad reference to a database table, and the second adds a missing legend label for a plot.

### CODE REVIEW

File: jwql/website/apps/jwql/monitor_pages/monitor_dark_bokeh.py [hunk 1]
```diff
@@ -651,8 +651,10 @@ def create_plot(self):
             # for each amp, we can get away with using use_amp=1 at the moment.
             if '5' in self.mean_dark:
                 use_amp = '5'
+                legend_label = 'Full aperture'
             else:
                 use_amp = '1'
+                legend_label = 'Amp 1'
```
Comment (bhilbert4) [inline jwql/website/apps/jwql/monitor_pages/monitor_dark_bokeh.py:657]: This still needs to be tested on the test server.

Comment (bhilbert4) [reply jwql/website/apps/jwql/monitor_pages/monitor_dark_bokeh.py:657]: Done. Looks good.

**Comment by bhilbert4** [comment]
@mfixstsci Small PR that's ready for review. Thanks!

File: jwql/website/apps/jwql/monitor_pages/monitor_dark_bokeh.py (hunk 1 above)
Comment (mfixstsci) [reply jwql/website/apps/jwql/monitor_pages/monitor_dark_bokeh.py:657]: Thanks!

**Review by mfixstsci — APPROVED** [review]
Thanks @bhilbert4

**Pull request merged**